docs/source/test.rst

```diff
@@ -3,6 +3,50 @@ Testing with Titanoboa
 
 Titanoboa integrates natively with `pytest <https://docs.pytest.org/>`_ and `hypothesis <https://hypothesis.readthedocs.io/en/latest/quickstart.html>`_. Nothing special is needed to enable these, as the plugins for these packages will be loaded automatically. By default, isolation is enabled for tests - that is, any changes to the EVM state inside the test case will automatically be rolled back after the test case completes.
 
+Getting Started
+-----------------------
+
+To begin writing your smart contract tests with pytest, it is reccomended to create a `tests/` directory to contain your tests. You will also need an `__init__.py` file (empty) within your test directory.
```

Comment:
i don't think you need an `__init__.py` file

Reply:
AFAIK I think its needed to import fixtures from `conftest.py` but I'm not sure on this

Reply:
oh, i don't think you are supposed to import fixtures from conftest, the fixtures are directly available as arguments to test functions

Reply:
Oh TIL, I guess I need to update some tests

Reply:
> oh, i don't think you are supposed to import fixtures from conftest, the fixtures are directly available as arguments to test functions

But constants and other functions defined in conftest need to be imported manually. Am I right?
Also, I am compiling some examples here:
https://github.com/El-Ku/Vyper_Examples_With_TitanoBoa/tree/master/all_contracts

Reply:
right